### ISSUE TITLE
RunAnything plugin - Expose Context/RF built-in variables ?

### ISSUE THREAD

**Issue by Anon1988** [robotframework/RIDE#1491]
Hi there!

I'm trying to create a macro with VCS commands to save the tests suites in an automated way so that the QA team can create and save it's tests as needed; But I can't find a way to expose the suite path to the plugin.

Reading a bit I found that the ${CURDIR} is calculated at Execution time but since that's not exposed to the plugin I tried to debug how it's calculated with no luck. I also tried to see how RIDE TestRunner Class calculated the command used to execute the tests, but it seems it's already initialized that way.

With that said, I was wondering if there is a way to save,calculate  or expose the current directory being used to the plugin in a RF fashion, or perhaps a feasible work-around.

**Comment by yanne** [comment]
During test execution, ${CURDIR} is always equal to the directory where `pybot` command is issued.
In RIDE, it is equal to the directory where RIDE was started. 

Do you really need the ${CURDIR}, or is it enough to have the absolute path to the current test suite?

**Comment by Anon1988** [comment]
Yes, if possible, the absolute Path to the current Directory would be enough.
